src/PublicationCollection.js

```diff
@@ -101,6 +101,7 @@ var PublicationCollection = Backbone.Collection.extend({
    * Starts observing the reactive query for changes.
    */
   startObservingChanges() {
+    if (!this._reactiveQuery) return;
```

Comment:
Do we need another guard [here](https://github.com/mixmaxhq/backbone-publication/pull/23/files#diff-f39dff00f46e265e64407af9ac1db78dR53)?